package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -65,170 +51,182 @@ Citations for included algorithms and modules
-   *Biophys J* **88**, 1291–1299 (2005). doi: `10.1529/biophysj.104.052449`_
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. _`10.1529/biophysj.104.052449`: https://dx.doi.org/10.1529%2Fbiophysj.104.052449
+   Amadei1999
+   Leo-Macias2005
 
 If you use :meth:`~MDAnalysis.analysis.pca.PCA.cumulative_overlap` or
-:func:`~MDAnalysis.analysis.pca.cumulative_overlap` please cite [Yang2008]_ .
+:func:`~MDAnalysis.analysis.pca.cumulative_overlap` please cite
+:cite:p:`a-Yang2008`.
 
-.. [Yang2008] Yang, L., Song, G., Carriquiry, A. & Jernigan, R. L.
-   Close Correspondence between the Motions from Principal Component Analysis of Multiple HIV-1 Protease Structures and Elastic Network Modes.
-   *Structure* **16**, 321–330 (2008). doi: `10.1016/j.str.2007.12.011`_
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. _`10.1016/j.str.2007.12.011`: https://dx.doi.org/10.1016/j.str.2007.12.011
+   Yang2008
 
 If you use the Mean Squared Displacement analysis code in
-:mod:`MDAnalysis.analysis.msd` please cite [Calandri2011]_ and [Buyl2018]_.
-
-.. [Calandri2011] Calandrini, V., Pellegrini, E., Calligari, P., Hinsen, K., Kneller, G. R.
-   NMoldyn-Interfacing Spectroscopic Experiments, Molecular Dynamics Simulations and Models for Time Correlation Functions.
-   *Collect. SFN*, **12**, 201–232 (2011). doi: `10.1051/sfn/201112010`_
-
-.. _`10.1051/sfn/201112010`: https://doi.org/10.1051/sfn/201112010
+:mod:`MDAnalysis.analysis.msd` please cite :cite:p:`a-Calandrini2011` and
+:cite:p:`a-Buyl2018`.
 
-.. [Buyl2018] Buyl, P. tidynamics: A tiny package to compute the dynamics of stochastic and molecular simulations. Journal of Open Source Software,
-   3(28), 877 (2018). doi: `10.21105/joss.00877`_
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. _`10.21105/joss.00877`: https://doi.org/10.21105/joss.00877
+   Calandrini2011
+   Buyl2018
 
 If you calculate shape parameters using
 :meth:`~MDAnalysis.core.group.AtomGroup.shape_parameter`,
 :meth:`~MDAnalysis.core.group.ResidueGroup.shape_parameter`,
-:meth:`~MDAnalysis.core.group.SegmentGroup.shape_parameter`
-please cite [Dima2004a]_.
-
-.. [Dima2004a] Dima, R. I., & Thirumalai, D. (2004). Asymmetry
-   in the shapes of folded and denatured states of
-   proteins. *J Phys Chem B*, 108(21),
-   6564-6570. doi:`10.1021/jp037128y
-   <https://doi.org/10.1021/jp037128y>`_
-
-If you calculate asphericities using
+:meth:`~MDAnalysis.core.group.SegmentGroup.shape_parameter`,
+or if you calculate asphericities using
 :meth:`~MDAnalysis.core.group.AtomGroup.asphericity`,
 :meth:`~MDAnalysis.core.group.ResidueGroup.asphericity`,
-:meth:`~MDAnalysis.core.group.SegmentGroup.asphericity`
-please cite [Dima2004b]_.
+:meth:`~MDAnalysis.core.group.SegmentGroup.asphericity`,
+please cite :cite:p:`a-Dima2004`.
 
-.. [Dima2004b] Dima, R. I., & Thirumalai, D. (2004). Asymmetry
```

Comment:
The reference with key `[Dima2004a]` seems to be the same as the one here referred to as `[Dima2004b]`. I left one single reference with key `Dima2004` and grouped all the methods listed above and below.